app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is defined as a class variable, which means it is shared across all instances of `Animal`. This is correct if you want to keep track of all alive animals globally. However, ensure this is the intended behavior, as it might lead to unexpected results if not managed carefully.